src/frontend-pwa/src/components/Button/button.styles.ts

```diff
@@ -16,20 +16,22 @@ type ButtonProps = {
 
 const StyledButton = styled.button<ButtonProps>`
   ${typography.toString()}
-  background-color: #003366;
+  width: ${(props) => (props.size === 'sm' ? '100px' : props.size === 'md' ? '200px' : '300px')};
```

Comment:
I suggest using `em` units here, unless there is a specific reason for `px` values.

Reply:
I've made a note of this and will change them accordingly. I used specific pixel sizes at the moment to get the size I want for the home buttons. I think I will leave it until after the sprint review tomorrow, but I put it in the backlog to be changed next sprint. I just don't want to mess with my styling too much before the review.